docs/user-guide/global-configurations/scoped-variables.md

```diff
@@ -0,0 +1,168 @@
+| `selectors.attributeSelectors`                  | object | A map of attribute selectors to values                                         |
+| `selectors.attributeSelectors.<selector_key>`   | string | The key of the attribute selector, e.g., *ApplicationName*, *EnvName*, *ClusterName* |
+| `selectors.attributeSelectors.<selector_value>` | string | The value of the attribute selector                                            |
+
+
+Here's a truncated template containing the specification of two variables for your understanding:
+
+    apiVersion: devtron.ai/v1beta1
+    kind: Variable
+    spec:
+
+    # First example of a variable
+    - name: DB_URL 
+    shortDescription: My application's customers are stored
+    notes: The DB is a MySQL DB running version 7.0. The DB contains confidential information.
+    isSensitive: true 
+    values: 
+        - category: Global 
+        value: mysql.example.com 
+
+    # Second example of a variable
+    - name: DB_Name
+    shortDescription: My database name to recognize the DB
+    notes: NA
+    isSensitive: false 
+    values: 
+        - category: Global 
+        value: Devtron 
+        - category: ApplicationEnv 
+        value: app1-p
+        selectors:
+            attributeSelectors:
+            ApplicationName: MyFirstApplication
+            EnvName: prod
+
+### Upload the Template
+
+1. Once you save the YAML file, go back to the screen where you downloaded the template.
+
+2. Use the file uploader utility to upload your YAML file.
+
+    ![Figure 2: Uploading the Template](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/scoped-variables/file-uploader.jpg)
+
+3. The content of the file will be uploaded for you to review and edit. Click **Review Changes**.
+
+    ![Figure 3: Reviewing the YAML file](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/scoped-variables/file-preview.jpg)
+
+4. You may check the changes between the last saved file and the current one before clicking **Save**.
+
+    ![Figure 4: Saving the file](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/scoped-variables/save.jpg)
+
+5. Click the **Variable List** tab to view the variables. Check the [How to Use a Scoped Variable](#how-to-use-a-scoped-variable) section to know more.
+
+    ![Figure 5: List of Variables](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/scoped-variables/variable-list.jpg)
+
+---
+
+## How to Edit an Exiting Scoped Variable
+
+Only a super-admin can edit existing scoped variables.
+
+**Option 1**: Directly edit using the UI
+
+![Figure 6: Editing from UI](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/scoped-variables/edit.jpg)
+
+**Option 2**: Reupload the updated YAML file
```

Comment:
A message that this will replace the variable file, so if any varaible existed before and not in latest one will be deleted / lost